### PR TITLE
Feature/#24(review after Feature/#19)

### PR DESCRIPTION
**Pull request by mariakravchenko**
- [x] eslint
- [x] editorConfig
- [x] prettier
- [x] pre-push/commit hook with prettier auto fixing

### CODE REVIEW

**Comment by mariakravchenko** [comment]
config we can use to configure your IDE(my VSC settings.sjon in .vscode)
```
{
  "editor.tabSize": 2,
  "editor.insertSpaces": true,
  "files.exclude": {
    "**/.git": true,
    "**/*.js": {
      "when": "$(basename).ts"
    },
    "**/*.js.map": {
      "when": "$(basename)"
    }
  },
  "eslint.autoFixOnSave": true,
  "eslint.validate": [
    "javascript",
    "javascriptreact",
    {
      "language": "typescriptreact",
      "autoFix": true
    }
  ],
  "editor.formatOnSave": true,
  "[javascript]": {
    "editor.formatOnSave": false,
  },
  "[typescript]": {
    "editor.formatOnSave": false,
  },
  "prettier.singleQuote": true
}

```

required plugins
esbenp.prettier-vscode
editorconfig.editorconfig
dbaeumer.vscode-eslint

File: package.json [hunk 1]
```diff
@@ -1,13 +1,35 @@
...
+    "eslint-config-airbnb-base": "^14.0.0",
+    "eslint-config-prettier": "^6.4.0",
+    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-prettier": "^3.1.1",
+    "babel-eslint": "10.0.3",
+    "nodemon": "1.19.2",
+    "prettier": "^1.18.2",
+    "eslint-import-resolver-webpack": "0.11.1",
+    "eslint-plugin-jsx-a11y": "6.2.3",
+    "eslint-plugin-react": "7.15.0"
```
Comment (aliaksei-lyzo) [inline package.json]: Though it's controversial, but I personally prefer no ^ in versions. We've had a case, when updated minor version on one package actually broke something, so I prefer to update packages manually...

File: packages/client/src/components/Home/Home.jsx [hunk 2]
```diff
@@ -1,8 +1,14 @@
 import React from 'react';
+import { Helmet } from 'react-helmet-async';
 
-const Home = () => <div>HOME</div>;
+const Home = () => [
+  <Helmet>
+    <title>Home</title>
+  </Helmet>,
+  <div>HOME</div>,
+];
```
Comment (aliaksei-lyzo) [inline packages/client/src/components/Home/Home.jsx]: imo it's better to use React.Fragment, as:
1) We won't need commas, hence more readability
2) We won't need keys

File: packages/client/src/index.html [hunk 3]
```diff
@@ -1,13 +1,18 @@
...
-    </head>
-    <body>
-    <div id="app"></div>
-    </body>
-</html>
\ No newline at end of file
+
+<head>
+  <meta charset="utf-8" />
+  <title>MCC React</title>
```
Comment (aliaksei-lyzo) [inline packages/client/src/index.html]: I think the title may be changed to Storm the Front :)

**Pull request merged**